Mathlib/Topology/Compactness/PseudometrizableLindelof.lean

```diff
@@ -0,0 +1,33 @@
+/-
+Copyright (c) 2023 Josha Dekker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Josha Dekker
+-/
+
+import Mathlib.Topology.Metrizable.Basic
+import Mathlib.Topology.Compactness.Lindelof
+
+/-!
+# Second-countability of pseduometrizable Lindelöf spaces
+
+Factored out from `Mathlib.Topology.Compactness.Lindelof`
+to avoid circular dependencies.
+-/
+
+variable {X : Type*} [TopologicalSpace X]
+
+open Set Filter Topology TopologicalSpace
+
+instance SecondCountableTopology.ofPseudoMetrizableSpaceLindelofSpace [PseudoMetrizableSpace X]
+    [LindelofSpace X] : SecondCountableTopology X := by
+  letI : PseudoMetricSpace X := TopologicalSpace.pseudoMetrizableSpacePseudoMetric X
+  have h_dense (ε) (hpos: ε > 0): ∃ s : Set X, s.Countable ∧ ∀ x, ∃ y ∈ s, dist x y ≤ ε := by
```

Comment:
In general, using < or $\leq$ is preferred over the other direction. (mathlib3 even had a linter for this; it's an open discussion if adding this linter in mathlib4 is a good idea.) Also, please add a space around the colon --- all in all, I'd write the following:
```suggestion
  have h_dense (ε) (hpos: 0 < ε) : ∃ s : Set X, s.Countable ∧ ∀ x, ∃ y ∈ s, dist x y ≤ ε := by